features/steps/givens/attributes.py

```diff
@@ -135,14 +146,23 @@ def step_impl(context, file_or_model, field, values):
 def step_impl(context, inst, attribute, tail="single"):
     yield ValidationOutcome(instance_id=getattr(inst, attribute, None), severity = OutcomeSeverity.PASSED)
 
-@gherkin_ifc.step("Its {attribute} attribute {condition} with {prefix}")
+@gherkin_ifc.step("Its {attribute} attribute {condition:start_or_not_start} with {prefix}")
+@gherkin_ifc.step("Its {attribute} attribute {condition:start_or_not_start} with {literal} {prefix}")
```

Comment:
The phrase with `{literal}` does not appear to be used. Was that an earlier experiment?

Reply:
Yes, indeed. There was some discussion on how strict we should be in our matching, I did some experiments with pattern matching (e.g. `Pset_Name`' vs `Pset Name` vs `pset Name`). I put it on hold until a conclusion was made. Let's try to get to one this afternoon.

Reply:
Perfect, add it to the agenda so that we're all prepared :)